Dockerfile

```diff
@@ -120,6 +120,12 @@ print("\n".join(str(v) for v in solcx.get_installed_solc_versions()))
   exit 1
 }
 
+# Show installed packages and versions
+pip list
+
+# Show outdated packages
+pip list -o
```

Comment:
do we need this part?

Reply:
not that important, removed in https://github.com/Consensys/mythril/pull/1869/commits/771171e5a29db828c60f274e5df50098c7d402cd 

keeping the plain `pip list` call as that helps to see what is actually installed in the container during build time.